modules/oadp-installing-oadp-rosa-sts.adoc

```diff
@@ -11,7 +11,9 @@ AWS Security Token Service (AWS STS) is a global web service that provides short
 
 [IMPORTANT]
 ====
-Restic is not supported in the OADP on ROSA with AWS STS environment. Ensure the Restic service is disabled. Use native snapshots to backup volumes. See _Known Issues_ for more information.
+Restic and Kopia are not supported in the OADP on ROSA with AWS STS environment. Make sure that the Restic service is disabled in the `DataProtectionApplication` resource.
```

Comment:
Restic and Kopia >> FS (File System) Restic and Kopia backups
This line:
Make sure that the Restic service is disabled in the `DataProtectionApplication` resource
Is misleading, you can omit it, because Restic and Kopia can be enable in DPA but the important thing is not to use it in the backup itself by setting: backup.spec.defaultVolumesToFsBackup: true.